### PR TITLE
Update i18n.md with information about Unicode isolation marks

### PR DESCRIPTION
**Pull request by alyavasilyeva**
Provide information about possible formatting issues when using placeables and how to solve them.

Context: https://github.com/grammyjs/i18n/issues/47

### CODE REVIEW

File: site/docs/plugins/i18n.md
Comment (KnorpelSenf) [inline site/docs/plugins/i18n.md:620]: I think it's enough to use a yellow warning box instead of a new section:
```suggestion
:::warn Possible formatting issues
```

File: site/docs/plugins/i18n.md
Comment (KnorpelSenf) [inline site/docs/plugins/i18n.md:633]:
```suggestion
:::

```
closes the box again

File: site/docs/plugins/i18n.md
Comment (KnorpelSenf) [inline site/docs/plugins/i18n.md:621]:
```suggestion
```
style nitpick

**Review by KnorpelSenf — APPROVED** [review]
Thank you so much!

**Comment by KnorpelSenf** [comment]
@alyavasilyeva just some quick info about what's gonna happen next: 
1. @roj1512 (website maintainer) will do a final review 
2. Once he approves, you're done here and you don't have to do anything anymore
3. Translators will take over and sync up your changes across all languages
4. This will be merged

File: site/docs/plugins/i18n.md
Comment (roj1512) [inline site/docs/plugins/i18n.md:620]:
```suggestion
::: warning Potential Formatting Issues
```

File: site/docs/plugins/i18n.md
Comment (roj1512) [inline site/docs/plugins/i18n.md:625]:
```suggestion
To fix this, use the following options:
```

**Review by roj1512 — APPROVED** [review]
Thank you!

**Comment by niusia-ua** [comment]
Don't we first merge third-party contributions to the main branch and then create PR with translations?

**Comment by KnorpelSenf** [comment]
> Don't we first merge third-party contributions to the main branch and then create PR with translations?

We used to do that so that you don't have to add a new remote, but I think we can stop doing that because GitHub advertises the changes in this pull request via a remote branch, doesn't it? So you can actually view the diff locally or even merge without cloning the fork. Can you try?

**Comment by niusia-ua** [comment]
I don't know what you're talking about. As far as I know, I need to make a fork of a fork and make changes from my fork to the fork.

**Comment by KnorpelSenf** [comment]
You never had to fork a fork, you always could just clone the fork and push to the branch. You automatically get write access to a branch of the third-party fork as soon as the branch is used to open a pull request.

However, you can actually just keep the main copy of this repository (no forks). You can then follow https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally?platform=linux&tool=cli#modifying-an-inactive-pull-request-locally to see the changes of a pull request. But it seems like those refs are read-only, so perhaps it is not that useful after all. We may want to keep on merging to a temporary branch and performing translations there.

File: site/docs/plugins/i18n.md
Comment (quadratz) [inline site/docs/plugins/i18n.md:621]: This is the first time I've heard this term. This link may help readers who are also new to the topic.
```suggestion
By default, Fluent uses [Unicode isolation](https://github.com/projectfluent/fluent.js/wiki/Unicode-Isolation) marks for interpolations. 
```

**Review by quadratz — CHANGES_REQUESTED** [review]

**Review by quadratz — CHANGES_REQUESTED** [review]
According to https://github.com/projectfluent/fluent.js/wiki/Unicode-Isolation#what-does-it-look-like

> ....
> Assume the following Fluent message which receives a URL as an argument from the app:
> ```fluent
> # In English
> privacy-more = Visit {$PRIVACY_URL} to learn more.
> # In Arabic (courtesy of Google Translate)
> privacy-more = تفضل بزيارة {$PRIVACY_URL} لمعرفة المزيد.
> ```
> Let the URL end with a forward slash, like so:
> ```json
> {"PRIVACY_URL": "https://www.mozilla.org/privacy/"}
> ```
> Without isolation, the rendered result shows the trailing slash on the wrong side of the URL:
> 
>> تفضل بزيارة https://www.mozilla.org/privacy/ لمعرفة المزيد. ❌
> 
> With isolation, the forward slash is displayed correctly:
> 
>> تفضل بزيارة ⁨https://www.mozilla.org/privacy/⁩ لمعرفة المزيد. ✅
> 
>
> .... If `useIsolation` is set to `false`, there's a risk that some translations won't look correctly in bidi scenarios. For RTL languages (Arabic, Hebrew, Persian and more)….

Based on the information above, are you certain it won't cause other issues if we turn off this feature? It might be beneficial to inform the reader about this effect so they are aware of the potential consequences.

**Comment by niusia-ua** [comment]
> You never had to fork a fork, you always could just clone the fork and push to the branch. You automatically get write access to a branch of the third-party fork as soon as the branch is used to open a pull request.

Wow, it really works! But it's quite inconvenient to make a clone of the website repository for each third-party contribution. Of course, third-party contributions are quite rare compared to our own, but it's still not convenient.

I think it would be better to accept third-party contributions in a temporary branch that translators can easily work with.

**Comment by KnorpelSenf** [comment]
Should we merge this into a temporary branch for the remaining translations?

**Comment by niusia-ua** [comment]
@KnorpelSenf I think so.

**Review by deptyped — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
Thanks @alyavasilyeva! These changes will be merged via #1058 after translation is done.

**Pull request closed without merging**
